pages/blog/integrating-mailchimp-with-your-nodejs-app.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: "Integrating MailChimp with your Node.js App"
+date: "2021-06-01"
```

Comment:
Keep an eye on this, based on how long this PR hangs out there. You'll want to make sure it pops up to the top of the list when it is eventually published.

Reply:
definitely! updated the date for today for the time being